vortex-array/src/compute/list.rs

```diff
@@ -329,11 +307,10 @@ mod tests {
         Some("a"),
         bool_array(vec![false, true, true], None)
     )]
-    // Case 2: list(utf8?) with NULL search scalar
```

Comment:
Haha well this doesn't really solve it, like how does this inconsistency not bother you! Update a comment instead of deleting it please 🙏

Reply:
i removed them all